apps/system/js/bluetooth_transfer.js

```diff
@@ -530,6 +538,38 @@ var BluetoothTransfer = {
       detail: details}));
   },
 
+  /**
+   * Handle related system notification message operation after reboot.
+   * Need close notification message by ourself.
+   *
+   * @param {Object} message system notification message
+   */
+  handleSystemMessageNotification: function(message) {
+    this.debug('Received system message: ' + JSON.stringify(message));
+    var evt = {
+      fileName: message.body,
+      contentType: message.data.contentType
+    };
+    this.openReceivedFile(evt);
+    this.closeSystemMessageNotification(message);
+  },
+
+  /**
+   * Close the system notification message.
+   *
+   * @param {Object} message system notification message
+   */
+  closeSystemMessageNotification: function(msg) {
```

Comment:
nit: `message` since it's defined in your doc.(message system notification message)